### DISCUSSION TITLE
Altered Beast US shows as Japan

### DISCUSSION THREAD

**Discussion by PsychoFox11** [sanni/cartreader#712] [Need help]
I don't mean to be the only one making noise here! If there's a better place to ask, please let me know. Maybe this is normal. I have a US copy of Altered Beast, from the original bundle I got in '89. It gets identified as Rev A of the Japanese title for it. It was an early console (I remember the box even showed Altered Beast in a MD shell in the Genesis on the front). Though if that were the case, I'd think it would be a combo US/Japan version, if they just stuck MD boards in Genesis shells to start with, but it just shows as Juuouki (Japan) (Rev A). I did search all around online best I could before asking here as I thought it was may be normal, but wasn't finding any info. The cart name shows as 'bL'.

Anyone know if this is an issue with the ID or just a regular thing?

**Comment by sanni** [comment]
If the CRC32 of your ROM dump matches Juuouki (Japan) (Rev A) then it seems like you got a Japanese PCB inside your US Genesis cartridge. 
If it was standard back then for the first waves of Altered Beast then it probably should be renamed to (Japan, USA) by no-intro. I don't know how exactly they handle such cases. Probably needs a second source confirmation.

**Comment by PsyK0p4T** [comment]
can you show us a pic of the pcb please ?
"Altered Beast" from USA and Europe both have a rom chip with "MPR-12538".
I would expect something different for Juuouki rev0 and revA.

**Reply by PsychoFox11** [reply]
I definitely will tomorrow. I'm in the middle of a move and my screwdrivers are packed up and at the new place. I'll be able to grab them tomorrow though.

**Comment by PsychoFox11** [comment]
I'm still going to follow up with my Altered Beast PCB pics tomorrow, but I ran into another one.
My US copy of Revenge of Shinobi, gets identified and labeled as The Super Shinobi (Japan)

For this one, I get a CRC32 of 5C7E5EA6, which seems it should match up to Revenge Of Shinobi, The (USA, Europe) (REV 00) according to this: https://gamehacking.org/game/116292

Not sure if I should make a seperate thread for this one, seemed related (US game Identified as JPN).

**Reply by PsyK0p4T** [reply]
Personnaly I would more trust datomatic data. They are giving :
Revenge of Shinobi, The (USA, Europe) : CRC 05f27994, and chip MPR-12930
Revenge of Shinobi, The (USA, Europe) (Rev A) : CRC fe91ab7e, and chip MPR-12930A
Revenge of Shinobi, The (USA, Europe) (Rev B) : CRC 4d35ebe4 , and chip MPR-12930B
Super Shinobi, The (Japan) : CRC 5c7e5ea6 , and chip MPR-12675

Can you open this cart too to check ? Pics of the outside and pcb would be interesting.

**Reply by PsychoFox11** [reply]
Just making sure you saw the pics I posted below this since it wasn't a direct reply. Haven't communicated on Github a ton. Do those mean anything to you?

**Comment by PsychoFox11** [comment]
Ok here ya go @PsyK0p4T - This was my original Altered Beast with my console from fall '89. I got Revenge of Shinobi when it was very new as well, from a friend of my dad's that sold/traded games on the side. It's obviously been heavily used. I believe it is one that has Spider-Man, but Batman has been replaced with the man-bat creature, so like the in-between version before they were both replaced. At least that's what I played, and I can't remember having more than one copy of this, but I've had a few friends dump old collections on me and am not 100% certain it is the same cart I got as a kid, but am like 90% sure hah. I'm sure about Altered Beast.
Here's a lot of pics! In the board pics, Altered Beast is on top. Used flash on the backs to see the text as the wording is a little different and I'm not sure if that tells anything. One's made in Japan, one in Taiwan.
![fronts](https://user-images.githubusercontent.com/5882157/219979026-0247bf0b-f955-497c-9a5c-fa5dfb2c5558.png)
![backs](https://user-images.githubusercontent.com/5882157/219979030-21642dda-b6dc-47a5-ab82-cbd12f39d4d3.png)
![tops](https://user-images.githubusercontent.com/5882157/219979038-7d75c789-09b3-49a4-a194-988c292a8ae0.png)
![boardfronts](https://user-images.githubusercontent.com/5882157/219979047-0462b20a-1f76-4315-af2a-a5a173890610.png)
![boardbacks](https://user-images.githubusercontent.com/5882157/219979054-074bc839-d2e5-418b-a409-82bff1a3c00c.png)

**Reply by PsyK0p4T** [reply]
Hi, and sorry for the late answer ! Thanks a lot for these pics.

About "Super Shinobi", in fact, you got the MPR-12675 chip inside, which was used in japan. So at least, no surprise to get a rom matching the JPN entry in database. The cart is USA for sure, but then it's hard to say if the pcb was swapped at some point... if you can't be totally sure of its origin, or original first owner.

About Altered Beast, this one is for sure curious. Even more if you are the cart's owner since the very beginning, and if it was sold like this in the US... It would mean that the very first batch sent to the US contained the JPN pcb ? Maybe this should be shared with No-intro datters. So your dump got the CRC : b2233e87  right ?

Too bad I can't find any pic of "Juuouki" (rev0 or revA) PCB to confirm the chip MPR codes of the carts sold in japan... But nearly sure it's the same as yours.

**Reply by PsychoFox11** [reply]
yeah, I'll ask in No-Intro. Checking out this Revenge of Shinobi, it has to be my original, I don't have 2 copies, and it isn't a game I ever would have gotten rid of. I remember Gamepro publishing a cart-swap trick where you put any other game in first, then hot swap Revenge of Shinobi, and you get the Super Shinobi title screen. Mine did that, but I think it may work on any version. I'm guessing it does some region check that gets bypassed by the hot swap.

And yeah that was the CRC32 of my dump.
